pkg/internal/cluster/providers/docker/provision.go

```diff
@@ -56,6 +57,10 @@ func provision(cluster string, cfg *config.Cluster) error {
 	if isHA {
 		apiServerPort = 0              // replaced with a random port
 		apiServerAddress = "127.0.0.1" // only the LB needs to be non-local
+		if isIPv6 {
+			apiServerAddress = "::1" // only the LB needs to be non-local
```

Comment:
Have you tested this? We were not doing this before as far as I could tell.

Reply:
will test it and report back